src/library/scala/collection/immutable/Vector.scala

```diff
@@ -955,7 +954,7 @@ private[immutable] trait VectorPointer[T] {
 
     private[immutable] final def copyOf(a: Array[AnyRef]) = {
```

Comment:
Minor: I think that the signature of `copyLeft`, `copyRight`, `copyRange` and `copyOf` should be the same (except for the name, of course), so maybe consider removing the `[immutable] final` modifiers here (or add them to the rest also)